### DISCUSSION TITLE
【开源项目自荐】基于angular的可视化编辑器

### DISCUSSION THREAD

**Discussion by bojue** [taowen/awesome-lowcode#135] [Show and tell]
仓库地址：[Github](https://github.com/bojue/Web-Editor)

在线预览：[在线预览](https://aistore.yichuansoft.com/dashboard/#/develop?project=1&page=2)

产品基于angular开发的大屏编辑器工具，已经应用到商业产品的编辑器开源。

##  产品截图

![image](https://github.com/taowen/awesome-lowcode/assets/14350577/71e93401-0bf6-4e70-8c35-8f142671f434)

**Comment by LowCoderGitHub** [comment]
同学可以到低代码行业交流平台上发贴哦 https://www.lowcoder.com.cn/
